R/zzz.R

```diff
@@ -60,6 +60,8 @@ can_charToRaw <- function(x) {
 
 stp <- function(...) stop(..., call. = FALSE)
 check_cassette_name <- function(x) {
+  if (length(x) != 1) stp("cassette name must be a single string")
```

Comment:
Not really related but I noticed that this was missing, so `use_cassette(letters)` generated an uninformative error.

Reply:
thanks!